NeedleFoundation.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'NeedleFoundation'
-  s.version          = '0.16.2.2'
+  s.version          = '0.17.0'
```

Comment:
Is this version bump required for you? Releasing a new version is a bit more involved than simply changing this variable. For example 0.16.2.2 hasn't made it to Homebrew yet with an open PR pending.

The rest of the PR looks good to me if CI's happy.

Reply:
Personally, I don't care about the version bump... but _technically_ this is a breaking change since anyone still building for iOS 8 (pretty unlikely) will no longer be able to use Needle as far as I know. If this is not a concern, then I really don't care what the version number is.

Thoughts?

Reply:
Yea your reasoning makes sense. In practice I doubt this is a huge breaking change that is worth the trouble of a version bump. (It's troublesome mostly because we are in middle of a tricky one right now).

So I say go ahead and revert the version bump and I can merge this for us :)